### ISSUE TITLE
make_eml examples validation fail

### ISSUE THREAD

**Issue by earnaud** [EDIorg/EMLassemblyline#54]
Hi,

I tested the `make_eml()` first example workflow to check how this function shall behave. Here is the full output:
```
Browse[1]> file.copy(
+   from = system.file('/examples/pkg_260', package = 'EMLassemblyline'),
+   to = tempdir(),
+   recursive = TRUE
+ )
[1] TRUE
Browse[1]> # Set working directory
Browse[1]> setwd(paste0(tempdir(), '/pkg_260'))
Browse[1]> # Make EML (for data package with data tables)
Browse[1]> make_eml(
+   path = './metadata_templates',
+   data.path = './data_objects',
+   eml.path = './eml',
+   dataset.title = 'Sphagnum and Vascular Plant Decomposition under Increasing Nitrogen Additions',
+   temporal.coverage = c('2014-05-01', '2015-10-31'),
+   maintenance.description = 'completed',
+   data.table = c('decomp.csv', 'nitrogen.csv'),
+   data.table.description = c('Decomposition data', 'Nitrogen data'),
+   user.id = 'csmith',
+   user.domain = 'EDI',
+   package.id = 'edi.260.1'
+ )
Making EML ...
Creating nodes ...
<eml>
  <access>
  <dataset>
    <title>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <creator>
    <associatedParty>
    <associatedParty>
    <associatedParty>
    <associatedParty>
    <pubDate>
    <abstract>
    <keywordSet>
    <keyword>
    <keyword>
    <keyword>
    <keyword>
    <keyword>
    <keyword>
    <keywordThesaurus>
   <additionalInfo>
    <intellectualRights>
    <coverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <geographicCoverage>
      <temporalCoverage>
      <taxonomicCoverage>
    <maintenance>
    <contact>
    <contact>
    <contact>
    <methods>
    <project>
      <relatedProject>
      <relatedProject>
      <relatedProject>
      <relatedProject>
    <dataTable> (decomp.csv)
    <dataTable> (nitrogen.csv)
    <additionalMetadata>
  <additionalMetadata>
</eml>
Writing EML (edi.260.1.xml)
Validating EML
EML validaton failed. See warnings for details.
Done.
```
What's more, the warnings couldn't be reached with the `warnings()` function:
```
Browse[1]> warnings()
Browse[1]> 
```
I also tried the other two examples (with other entities and both types). Still unsuccessfully. 
EDIT: I upgraded EMLassemblyline to fix_31b and the warnings were functional.
```
Warning messages:
1: Defaulting argument 'data.table.name' to 'data.table'. Consider providing a brief descriptive name for your file(s).
2: In eml_additional_validation(doc, encoding = encoding) :
  Document is invalid. Found the following errors:
 not all 'references' values match defined id attributes
```

**Comment by earnaud** [comment]
See dedicated [issue](https://github.com/earnaud/MetaShARK-v2/issues/26) in MetaShARK

**Comment by earnaud** [comment]
Also, running:
```
> EML::eml_validate("./eml/edi.260.1.xml")
[1] FALSE
attr(,"errors")
[1] "Element 'quote': This element is not expected." "Element 'quote': This element is not expected."
[3] "Element 'quote': This element is not expected." "Element 'quote': This element is not expected."
```
A colleague of mine told me this would be linked to a misformation in "Intellectual Rights"

**Comment by earnaud** [comment]
EDIT: I upgraded EMLassemblyline to `fix_31b` and the warnings were functional.
```
Warning messages:
1: Defaulting argument 'data.table.name' to 'data.table'. Consider providing a brief descriptive name for your file(s).
2: In eml_additional_validation(doc, encoding = encoding) :
  Document is invalid. Found the following errors:
 not all 'references' values match defined id attributes
```

**Comment by earnaud** [comment]
I would suggest providing a default value to the `data.table.description` argument, like: `data.table.description = data.table` . With this, it would technically give a description, although it would not contain any useful information.

**Comment by clnsmth** [comment]
Thanks for reporting this issue and these suggestions @earnaud.

I'll look into the error handling to make sure warnings and errors originating from `EML::eml_validate()` are accessible with `warnings()`.

Your colleague may be right about the intellectual rights quote characters. A quick fix is to use the "new" intellectual_rights.txt templates output by `template_core_metadata()`. This was a problem that began after upgrading the EML R package from 1.0.3 to 2.0.0, and that we patched by adding escape characters to quotes within the intellectual rights license. This is a problem when using quote characters within ".docx" versions of abstract, methods, and additional_info templates. It may also be a problem for ".txt" versions of these templates. I'll look for a solution to this (#55).

Providing default values, and relaxing input requirements, is something we'll do to accomodate more variation in user inputs (#34).

**Comment by clnsmth** [comment]
This has been fixed by ce7bb99de5dcf0a537869bf2ee453821cbd8dc17.
